### DISCUSSION TITLE
Equip Artifacts/Weapons toggle?

### DISCUSSION THREAD

**Discussion by TheLonelyDevil9** [Andrewthe13th/Inventory_Kamera#449] [Q&A]
What exactly do these options do? Does the scanner pick up what characters have the weapons and artifacts currently equipped? Does it account for only the weapons/artifacts that match the filters that are first being scanned for? The behavior isn't too clear, just thought I'd check for the exact behavior of these toggles.

Thank you for your help!

**Comment by Dyl-Aeon** [comment]
The Character scrape only pulls level, ascension, talent, constellation. No weapon or artifacts in the capture.
![image](https://github.com/Andrewthe13th/Inventory_Kamera/assets/55271102/ea210345-4dba-43ad-aabe-817ef9742111)

When you scrape artifacts or weapons, having this option toggled on will also show if the item is equipped and by who. Toggle off, the export will not show if the item is currently equipped.
![image](https://github.com/Andrewthe13th/Inventory_Kamera/assets/55271102/98f41820-c9bf-4773-a6b5-1aaa979a98de)
ie current location of item.

You're right though, the wording feels strange. My first read of it left me with the impression that it might unequip my gear.
